Docs/Migration-INI.md

```diff
@@ -0,0 +1,972 @@
+0=chkMEDDraw:FileSettingCheckBox
+
+[chkMEDDraw]
+Location=285,147
+Text=Enable DDWrapper for map editor
+ToolTip=Enables DirectDraw wrapper & emulation for map editor.@Turning this option on can help if you are encountering problems with editor viewport not displaying or being laggy. 
+EnabledFile0=Resources/Compatibility/DLL/ddwrapper.dll,Map Editor/ddraw32.dll,OverwriteOnMismatch
+EnabledFile1=Resources/Compatibility/Configs/aqrit.cfg,Map Editor/aqrit.cfg,KeepChanges
+DefaultValue=false
+SettingSection=Video
+SettingKey=UseDDWrapperForMapEditor
+```
+
+2. **OPTIONAL (YR+Phobos)** Add sections:
+
+```ini
+[GameOptionsPanelExtraControls]
+; Only available with Phobos
+0=chkTooltipsExtra:SettingCheckBox
+1=chkPrioritySelection:SettingCheckBox
+2=chkBuildingPlacement:SettingCheckBox
+
+[chkTooltipsExtra]
+Location=24,151, ;12,151
+Text=Sidebar Tooltip Descriptions
+ToolTip=Enables additional information in sidebar tooltips.
+DefaultValue=true
+ParentCheckBoxName=chkTooltips
+ParentCheckBoxRequiredValue=true
+SettingSection=Phobos
+SettingKey=ToolTipDescriptions
+
+[chkPrioritySelection]
+Location=242,54
+Text=Mass Selection Filtering
+ToolTip=If enabled, non-combat units are not selected if mass-selecting together with combat units.
+DefaultValue=false
+SettingSection=Phobos
+SettingKey=PrioritySelectionFiltering
+
+[chkBuildingPlacement]
+Location=242,78
+Text=Show Building Placement Preview
+ToolTip=If enabled, shows a preview image of the building when placing it.
+DefaultValue=false
+SettingSection=Phobos
+SettingKey=ShowBuildingPlacementPreview
+```
+
+3. Add sections:
+
+```ini
+[lblPlayerName]
+Location=12,195
+
+[tbPlayerName]
+Location=113,193
+
+[lblNotice]
+Location=12,220
+
+[btnConfigureHotkeys]
+Location=12,290
+
+[chkDisablePrivateMessagePopup]
+Location=12,138
+Text=Disable private message pop-ups
+
+[chkAllowGameInvitesFromFriendsOnly]
+Location=276,68
+Text=Only receive game invitations@from friends
+
+[lblAllPrivateMessagesFrom]
+Location=276,138
+
+[ddAllowPrivateMessagesFrom]
+Location=470,137
+
+[gameListPanel]
+Location=0,200
+```
+
+## Create new `PlayerExtraOptionsPanel.ini`
+
+A new panel that allows for convenient match setup has been added in the
+game lobby.
+
+```ini
+[btnClose]
+Location=220,0
+Size=18,18
+
+[lblHeader]
+Location=12,6
+
+[chkBoxForceRandomSides]
+Location=12,28
+
+[chkBoxForceRandomColors]
+Location=12,50
+
+[chkBoxForceRandomTeams]
+Location=12,72
+
+[chkBoxForceRandomStarts]
+Location=12,94
+
+[chkBoxUseTeamStartMappings]
+Location=12,130
+
+[btnHelp]
+Location=160,130
+
+[lblPreset]
+Location=12,156
+
+[ddTeamStartMappingPreset]
+Size=157,21
+Location=65,154
+
+[teamStartMappingsPanel]
+Location=12,189
+```
+
+## Appendix
+
+For completion's sake, below are additional steps required for a complete
+migration (beyond INI changes) to client version [2.11.0.0][client].
+
+### Replace binary files
+
+1. Replace contents of `Resources/Binaries` and `Resources/Compatibility` with

```

Comment:
Also, apart from`Resources/Binaries` folder, `Resources/client{dx, ogl, xna}.exe` are also a part of .NET 4.8 client. 

I think we can use the following tree view to better illustrate the copy process? 
```txt
Compiled
└── Ares
    └── Resources # override the `Resources` folder to update the client binaries
        ├── Binaries  # this folder contains partial .NET 4.8 client files
        ├── BinariesNET8 # this folder contains .NET 8.0 client files, where modders can either delete it, or keep it for an experimental cross-platform support
        ├── clientdx.exe
        ├── clientdx.exe.config
        ├── clientdx.pdb # .pdb file contains debug symbols. It can be either deleted or retained.
        ├── clientogl.exe
        ├── clientogl.exe.config
        ├── clientogl.pdb
        ├── clientxna.exe
        ├── clientxna.exe.config
        └── clientxna.pdb
```